clients/venice-admin-tool/src/test/java/com/linkedin/venice/zk_paths.txt

```diff
@@ -0,0 +1,34 @@
+/venice-parent
+/venice-parent/storeConfigs
+/venice-parent/cluster1
+/venice-parent/cluster1/storeConfigs
+/venice-parent/cluster1/adminTopicMetadata
+/venice-parent/cluster1/adminTopicMetadata/file1
+/venice-parent/cluster1/adminTopicMetadata/file2/file3
+/venice-parent/cluster1/executionids
+/venice-parent/cluster1/ParentOfflinePushes
+/venice-parent/cluster1/routers
+/venice-parent/cluster1/StoreGraveyard
+/venice-parent/cluster1/Stores
+/venice-parent/cluster2
+/venice-parent/cluster2/storeConfigs
+/venice-parent/cluster2/adminTopicMetadata
+/venice-parent/cluster2/executionids
+/venice-parent/cluster2/executionids/file1
+/venice-parent/cluster2/executionids/file2/file3
+/venice-parent/cluster2/ParentOfflinePushes
+/venice-parent/cluster2/routers
+/venice-parent/cluster2/StoreGraveyard
+/venice-parent/cluster2/Stores
+/venice-parent/cluster3
+/venice-parent/cluster3/storeConfigs
+/venice-parent/cluster3/adminTopicMetadata
+/venice-parent/cluster3/adminTopicMetadata/file1
+/venice-parent/cluster3/adminTopicMetadata/file2/file3
```

Comment:
In these, can we replace `cluster3` with something else to remove some confusion?